projects/src/simbicycle.hh

```diff
@@ -161,6 +162,21 @@ void Bicycle<T, U, V>::update_kinematics() {
     m_pose.steer = steer;
 }
 
+template <typename T, typename U, typename V>
```

Comment:
I would prefer to do this after merging all the pending PRs as there are a number of dependencies.